contracts/src/periphery/cert/router/EigenDACertVerifierRouter.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierBase, IEigenDACertVerifier} from "src/periphery/cert/interfaces/IEigenDACertVerifier.sol";
+import {IEigenDACertVerifierRouter} from "src/periphery/cert/interfaces/IEigenDACertVerifierRouter.sol";
+import {OwnableUpgradeable} from "lib/openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+
+contract EigenDACertVerifierRouter is IEigenDACertVerifierRouter, OwnableUpgradeable {
+    /// @notice A mapping from an activation block number (ABN) to a cert verifier address.
+    mapping(uint32 => address) public certVerifiers;
+
+    /// @notice The list of Activation Block Numbers (ABNs) for the cert verifiers.
+    /// @dev The list is sorted in ascending order, and corresponds to the keys of the certVerifiers mapping.
+    uint32[] public certVerifierABNs;
```

Comment:
"sorted in ascending order" sounds like we actively maintain its order. Prob best to say "the list is guaranteed to be in increasing ABN order"?
And the guarantee comes from the fact that we only allow ABNs to be added past the last one.